compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/PositionUtil.java

```diff
@@ -60,7 +60,7 @@ static boolean withinBlock(LinePosition cursorPos, Location symbolPosition) {
     }
 
     static boolean withinRange(LineRange specifiedRange, Location nodePosition) {
-        if (!nodePosition.lineRange().filePath().equals(specifiedRange.filePath())) {
+        if (nodePosition == null || !nodePosition.lineRange().filePath().equals(specifiedRange.filePath())) {
```

Comment:
I thought to have this as a hack, for now, since this fix is urgent.

Reply:
I would say since we already have an issue created for nodePosition being null (if not let's create one), let's handle this fix here since this is a blocking issue.

Reply:
Yeah, we have one. 
We will roll back this change along with the fix for the cause.